### PR TITLE
adding scandir wheel for big sur

### PR DESCRIPTION
**Pull request by oliviadawd**
not too sure if this is what i need, but running these commands:
```
pip wheel --no-cache-dir --wheel-dir=./ scandir==1.10.0
pip wheel --no-cache-dir --wheel-dir=./ pathlib2==2.3.5
./rebuild-index.sh
```
yielded this wheel for me. still not sure if we want to upgrade the version in `.python-version`, but we can talk about that later. i'm hoping this PR fixes this error i've been getting when trying to run python tests locally:
```
 Failed to execute PEX file, missing macosx_11_1_x86_64-cp-27-cp27m compatible dependencies for:
                     pathlib2
                     scandir
```

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -101,6 +101,7 @@ <h1>Index of python-pkgs</h1>
     <a href="pytz-2017.2-py2.py3-none-any.whl">pytz-2017.2-py2.py3-none-any.whl</a><br/>
     <a href="requests-2.10.0-py2.py3-none-any.whl">requests-2.10.0-py2.py3-none-any.whl</a><br/>
     <a href="scandir-1.10.0-cp27-cp27m-macosx_10_15_x86_64.whl">scandir-1.10.0-cp27-cp27m-macosx_10_15_x86_64.whl</a><br/>
+    <a href="scandir-1.10.0-cp38-cp38-macosx_11_0_x86_64.whl">scandir-1.10.0-cp38-cp38-macosx_11_0_x86_64.whl</a><br/>
```
Comment (MasterDDT) [inline index.html:104]: Shouldnt there be a wheel for pathlib2? Did it not build one? Maybe was already there?

**Comment by MasterDDT** [comment]
I think we could just nuke the `.python-version` file. I dont think theres a good reason to force a specific py2 version...whatever py2 version (as long as its not crazy old like py2.7.0), you use to build the wheel, it should be compatible.

File: index.html (hunk 1 above)
Comment (oliviadawd) [reply index.html:104]: it did not D: i was confused about that too but i think pathlib2 is `pathlib2-2.3.5-py2.py3-none-any.whl` so it doesn't care about the os version?

**Review by MasterDDT — APPROVED** [review]

**Comment by oliviadawd** [comment]
> I think we could just nuke the `.python-version` file. I dont think theres a good reason to force a specific py2 version...whatever py2 version (as long as its not crazy old like py2.7.0), you use to build the wheel, it should be compatible.

kk should i just go ahead and remove it in this PR then?

**Comment by MasterDDT** [comment]
> > I think we could just nuke the `.python-version` file. I dont think theres a good reason to force a specific py2 version...whatever py2 version (as long as its not crazy old like py2.7.0), you use to build the wheel, it should be compatible.
> 
> kk should i just go ahead and remove it in this PR then?

Sure

**Comment by oliviadawd** [comment]
hmmm so to get this wheel i need to do a `pip install -r deploy/requirements.txt` right? looks like im getting this error now:
```
pip install -r deploy/requirements.txt
Looking in links: https://actioniq-oss.github.io/python-pkgs/index.html
ERROR: Could not find a version that satisfies the requirement hcvault==0.3.11
ERROR: No matching distribution found for hcvault==0.3.11
```

EDIT: scratch that looks like my python version was wrong

**Pull request merged**